apps/masterbots.ai/components/routes/browse/browse-chatbot-details.tsx

```
@@ -70,20 +84,16 @@ export default function BrowseChatbotDetails({
             >
               Chat with {chatbot?.name} &gt;
             </Link>
-            <div className="flex gap-4">
-              {/* <IconUpVote className="h-4 opacity-60" />
-                <span className="text-[#72C255]">1.2k</span>
-              <IconDownVote className="h-4 opacity-60" />
-                <span className="text-[#F42F53]">375</span> */}
-              {/* <span className="text-[#FAFAFA]">17</span>               */}
-            </div>
+            <div className="flex gap-4"></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)